networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -230,7 +232,7 @@ spec:
 <2> A node selector that specifies the nodes to apply the node network configuration policy to.
 <3> A new OVS bridge, separate from the default bridge used by OVN-Kubernetes for all cluster traffic.
 <4> A network device on the host system to associate with this new OVS bridge.
-<5> The name for the additional network from which traffic is forwarded to the OVS bridge. This additional network must match the name of the additional network that you define in the `netAttachDefName` field of a `NetworkAttachmentDefinition` object.
+<5> The name for the additional network from which traffic is forwarded to the OVS bridge. This additional network must match the name of the `metadata.name` field of the `NetworkAttachmentDefinition` object that defines the OVN-Kubernetes additional network.
```

Comment:
ditto; this is wrong, it's not the metadata.name attribute it must match - it is the spec.config.name attribute.